cookbook/recipes/workflows/advanced_flags.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+
+from flytekit.sdk.tasks import inputs
+from flytekit.sdk.types import Types
+from flytekit.sdk.workflow import workflow_class, Input, Output
+from flytekit.common.tasks.presto_task import SdkPrestoTask
+from flytekit.models.common import RawOutputDataConfig
+
+schema = Types.Schema([("a", Types.Integer), ("b", Types.String)])
+
+presto_task = SdkPrestoTask(
```

Comment:
Hmm, just realized.  This will be the first file in this cookbook series that won't run locally.  Perhaps we should change the task type?